tiler-server/start.sh

```diff
@@ -23,12 +23,12 @@ function purgeCache(){
     do
       aws s3 cp ${AWS_S3_BUCKET}/${tileFile} $workDir/${tileFile}
       # Purge
-      tegola --config=/opt/tegola_config/config.toml cache purge --tile-list=$workDir/${tileFile} --tile-name-format="/zxy" --min-zoom 0 --max-zoom 20  --overwrite
-      err=$?
-      if [[ $err != "0" ]]; then
-        #error
-        echo "tegola exited with error code $err"
-      fi
+      echo tegola --config=/opt/tegola_config/config.toml cache purge --tile-list=$workDir/${tileFile} --tile-name-format="/zxy" --min-zoom 0 --max-zoom 20  --overwrite
```

Comment:
This doesn't look right. There's an `echo` before the command

Reply:
Good catch, I will add that!!